src/extension/inpage-script/index.js

```diff
@@ -121,6 +121,13 @@ if (document) {
     },
     { capture: true }
   );
+  // Listen for webln events from the extension
+  // emit more events as received
+  window.addEventListener("message", (event) => {
+    if (event.source === window && event.data === "accountSelected") {
+      window.webln.emit("accountSelected");
```

Comment:
- should we check if webln exists first?
- also we should fire this event for window.nostr, if it exists.

Reply:
resolved